src/Plugin/QueueWorker/MessageDeletionWorker.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Drupal\message\Plugin\QueueWorker;
+
+use Drupal\Core\Entity\EntityTypeManagerInterface;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Drupal\Core\Queue\QueueWorkerBase;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Delete messages.
```

Comment:
Could you update the 2 docblock descriptions for these new queue workers? It wasn't immediately clear at a glance what the difference was.